Source/WebCore/svg/graphics/filters/SVGFilterGraph.h

```diff
@@ -77,17 +78,21 @@ class SVGFilterGraph {
 
     RefPtr<NodeType> getNamedNode(const AtomString& id) const
     {
-        if (id.isEmpty()) {
-            if (m_lastNode)
-                return m_lastNode;
-
-            return sourceGraphic();
+        if (!id.isEmpty()) {
+            auto sourceNodes = m_sourceNodes.find(id);
+            if (sourceNodes != m_sourceNodes.end())
+                return m_sourceNodes.get(id);
```

Comment:
`HashMap::find()` returns an iterator. So we usually name the return value `it` or `iter` or `iterator`. Also `find()` is a look-up and `get()` is another lookup. You can use something like this:

```
auto it = m_sourceNodes.find(id);
if (it != m_sourceNodes.end())
    return it->value.ptr();
```

Or simpler like this:

```
if (auto node = m_sourceNodes.get(id))
    return node;
```

Reply:
I will compile locally simpler version and confirm, it progresses and then push another update. :-)